docs/csharp/language-reference/keywords/string.md

```diff
@@ -50,6 +50,18 @@ string str = "test";
 char x = str[2];  // x = 's';
 ```
 
+In similar fashion, the [] operator can also be used for iterating over each character in a `string`:
+
+```csharp
+string str = "test";
+
+for (int i = 0; i < str.Length; i++)
+{
+  Console.Write(str[i] + " ");
+}
+// Output: test
```

Comment:
Letters should be separated by a white space in the output.

Reply:
D'oh! And I put the `+ " "` there purposefully, too. Thanks for the catch.